DB/database.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const uri =
+	'mongodb+srv://prakharvarshney:fantastic10@sparrow-cluster.7brzk.mongodb.net/sparrow-store?retryWrites=true&w=majority';
```

Comment:
Avoid using passwords as plain text instead of this you can add this in the .env file.